packages/devtools_app/lib/src/timeline/flutter/timeline_model.dart

```diff
@@ -0,0 +1,1099 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+import 'dart:collection';
```

Comment:
update copyright notice or update the diff to include the file this is based on.

Reply:
updated copyright